components/list/list-item-drag-mixin.js

```diff
@@ -1,20 +1,82 @@
+import { css, html } from 'lit-element/lit-element.js';
 import { announce } from '../../helpers/announce.js';
+import { dragActions } from './list-item-drag-handle.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
-import { html } from 'lit-element/lit-element.js';
 import { nothing } from 'lit-html';
 
-export const ListItemDragMixin = superclass => class extends superclass {
+export const dropLocation = Object.freeze({
+	above: 1,
+	below: 2,
+	first: 3,
+	last: 4,
+	shiftDown: 5,
+	shiftUp: 6,
+	void: 0
+});
```

Comment:
I am not 100% on these names:
- **Above**: Move the element above the dropTarget.
- **Below**: Move the element below the dropTarget
- **First**: Move the element to the first position of the list.
- **Last**: Move the element to the last position of the list.
- **ShiftDown**: Shift the position of an element down by 1.
- **ShiftUp**: Shift the position of an element up by 1.
- **Void**: No drop location. Used to hide both markers.

Reply:
This is so much better!